rules/Transform/Rector/MethodCall/MethodCallToMethodCallRector.php

```diff
@@ -97,25 +97,22 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        $isMethodCallCurrentClass = $this->isMethodCallCurrentClass($node);
         foreach ($this->methodCallsToMethodsCalls as $methodCallToMethodCall) {
-            if (! $node->var instanceof PropertyFetch && ! $isMethodCallCurrentClass) {
```

Comment:
MethodCall var can be non-propery fetch or Variable, eg: Funcall:

```php
run()->execute();
```

so the check is needed.